src/components/form-select/form-select.tsx

```diff
@@ -0,0 +1,34 @@
+import { Controller, useFormContext } from "react-hook-form";
+import { FormSelectProps } from "./form-select.types";
+import { Form, Select } from "antd";
+
+export const FormSelect = ({ name, label, ...props }: FormSelectProps) => {
+    const {
+        control,
+        formState: { errors, defaultValues },
+    } = useFormContext();
+
+    const errorMessage = errors[name]?.message;
+    const defaultValue = defaultValues?.[name] as string;
```

Comment:
fixed, but made it so that both the helperText and defaultValue could be either string or undefined (since just defaulting to "" has some unwanted side effects (like antd still rendering a helper text node for an empty string, and setting a default value (even as an empty string) when it wasn't initially defined for that field also doesn't seem really necessary)